### ISSUE TITLE
Last timestamp not included in netcdf

### ISSUE THREAD

**Issue by kitbenjamin** [peterkuma/cl2nc#2]
Hi,

I have several CL31 files (one example attached [here](https://github.com/peterkuma/cl2nc/files/6191205/cl2nc_issue.zip)) where I have found the last timestamp has not been included in the netcdf.

Netcdf created with:
```
cl2nc cl2nc_issue_example.DAT cl2nc_issue_example.nc
```

From *cl2nc_issue_example.DAT* you can see the last timestamp in the file is **2021-03-21 01:59:56**

However, after a quick inspection of the netcdf:
``` python
import netCDF4 as nc 
import datetime as dt

nc_data = nc.Dataset('cl2nc_issue_example.nc')

netcdf_timestamps = [dt.datetime.fromtimestamp(i) for i in nc_data['time'][:]]

max(netcdf_timestamps).strftime('%Y-%m-%d %H:%M:%S')
```
```
>> '2021-03-21 01:59:42'
```
i.e. the second last timestamp in the file.

This is with cl2nc version 3.3.0 and python version 3.8.3

**Comment by peterkuma** [comment]
Thank you for reporting this issue. It should be fixed in cl2nc 3.3.1. Can you please test?

**Comment by kitbenjamin** [comment]
Thanks @peterkuma - that has sorted it
